vyxal/elements.py

```diff
@@ -1023,6 +1023,17 @@ def gen():
     return gen()
 
 
+def depth(lhs, ctx):
+    """Element Þj
+    (lst) -> depth of a
+    """
+    get_depth = lambda d: isinstance(d, list) and max(map(get_depth, d)) + 1
+    depth = get_depth(lhs)
+    if not depth:
+        return 0
+    return depth
```

Comment:
This can be simpified
```suggestion
    return int(get_depth(lhs)
```